Archipelago.MultiClient.Net/Colors/Color.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
```

Comment:
Should this especially not have any depencies on System.Drawing

Reply:
Whoops, just default imports. Using the namespace but no types from it does not cause the issue we've seen anyway. (Although I will remove the unnecessary usings.)